.github/workflows/lint.yml

```
@@ -0,0 +1,107 @@
+          reqs=$(sel 'requirements.*\.txt$' | grep -v '^src/api/requirements\.txt$' || true)
+          [ -n "$reqs" ] && { requirements-txt-fixer $reqs || true; }
+          cw=$(sel '^src/cook-web/.*\.(js|jsx|ts|tsx|mjs|cjs|css|scss|md|mdx|json)$')
+          [ -n "$cw" ] && { src/cook-web/node_modules/.bin/prettier --write --config src/cook-web/.prettierrc --ignore-path src/cook-web/.prettierignore $cw || true; }
+          cwlint=$(sel '^src/cook-web/.*\.(js|jsx|ts|tsx)$' | sed 's|src/cook-web/||g')
+          [ -n "$cwlint" ] && { (cd src/cook-web && npm run lint-file -- $cwlint) || true; }
+          true
+
+      - name: Push auto-fixes back to the PR branch
+        # Only on same-repo PRs (fork PRs have no write access to the head repo).
+        if: github.event.pull_request.head.repo.full_name == github.repository
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)